src/compiler/diagnosticMessages.json

```diff
@@ -855,6 +855,10 @@
         "category": "Error",
         "code": 1259
     },
+    "Keyword must not contain escaped characters": {

```

Comment:
is there a reason some diagnistics end with a dot and others don't?

Reply:
Because they're written by different humans 🙃

Ending dot is standard, though